src/testLetterMulti.js

```diff
@@ -0,0 +1,28 @@
+const letterArray = ['|', 'd', '|', 'o', 'g']
+
+const adjustForLetterMulti = (array) => {
+  const adjustedArray = []
+  const datasheet = require('./datasheet.js').datasheet
+  for (let i = 0; i < array.length; i++) {
+    if (array[i] === '[' && array[i + 2] === ']') {
+      adjustedArray.push(array[i + 1])
+      adjustedArray.push(array[i + 1])
+    } else if (array[i] === '{' && array[i + 2] === '}') {
+      adjustedArray.push(array[i + 1])
+    } else if (Object.getOwnPropertyNames(datasheet).includes(array[i])) {
+      adjustedArray.push(array[i])
+    } else if (
+      (array[i] === ']' && array[i - 2] === '[') ||
```

Comment:
I was trying to figure out how to have conditions like this in one line, I sure didn't think of this.